cross-media-measurement/src/test/kotlin/org/wfanet/measurement/common/throttler/MinimumIntervalThrottlerTest.kt

```diff
@@ -36,7 +36,7 @@ import org.junit.runners.JUnit4
 @OptIn(ExperimentalCoroutinesApi::class) // For `runBlockingTest`.
 class MinimumIntervalThrottlerTest {
   @Test
-  fun onReady() = runBlocking<Unit> {
+  fun onReady() = runBlocking {
```

Comment:
The issue with stripping these out is that if an assertThat over containers/iterables is the last statement in the test, then the inferred return type won't be unit. Does that matter?